modules/virt-accessing-vmi-ssh.adoc

```diff
@@ -14,6 +14,8 @@ the `fedora-vm-ssh` service that forwards port 22 of the `<fedora-vm>` virtual
 machine to a port on the node:
 
 .Prerequisites
+* You must be working in the {product-title} project in which the virtual machine
```

Comment:
I don't think the product-title is necessary here, and it should be VMI for consistency. Suggest to shorten it to something like:
"You must be in the same project as the virtual machine instance."

Reply:
Changed as suggested.